analyses/oncoprint-landscape/01-map-to-sample_id.R

```diff
@@ -283,11 +295,10 @@ cnv_df <- cnv_df %>%
   inner_join(select(histologies_df,
                     Kids_First_Biospecimen_ID,
                     sample_id),
-             by = "Kids_First_Biospecimen_ID") %>%
-  filter(status != "uncallable") %>%
+             by = c("biospecimen_id"="Kids_First_Biospecimen_ID")) %>%
   mutate(Tumor_Sample_Barcode =  sample_id) %>%
   rename(Variant_Classification = status,
```

Comment:
per [this comment](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/1005#issuecomment-836671602), we need to separate out gains and amplifications here and only carry through amps to the oncopolot - can you update this?

Reply:
To give a little more context (and to interject with my opinion about _how_ to do this)... 

Currently we recode this in `02-plot-oncoprint.R`: https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/c0744576ee80914636443b01012168efc2da22a9/analyses/oncoprint-landscape/02-plot-oncoprint.R#L149

I would remove that from `02-plot-oncoprint.R` and move it into this script (`01-map-to-sample_id.R`), where you mutate `Variant_Classification`:

```R
dplyr::case_when(
  Variant_Classification == "loss" ~ "Del",
  Variant_Classification  == "amplification" ~ "Amp",
  TRUE ~ as.character(Variant_Classification)
)
```

And then filter with `Variant_Classification %in%  c("Del", "Amp")`

Reply:
Updated now